api/gauge_agg.md

```diff
@@ -0,0 +1,84 @@
+---
+api_name: gauge_agg
+api_category: hyperfunction
+api_experimental: true
+hyperfunction_toolkit: true
+hyperfunction_family: 'counter aggregation'
+hyperfunction_subfamily: FrequencyAggregate
+hyperfunction_type: accessor
+---
+
+# gauge_agg() <tag type="toolkit" content="Toolkit" /><tag type="experimental" content="Experimental" />
+Produces a `GaugeSummary` that can be used to accumulate gauge data for further
+calculations. 
+```sql
+gauge_agg (
+    ts TIMESTAMPTZ,
+    value DOUBLE PRECISION
+) RETURNS GaugeSummary
+```
+
+<highlight type="warning">
+Experimental features could have bugs. They might not be backwards compatible,
+and could be removed in future releases. Use these features at your own risk,
+and do not use any experimental features in production.
+</highlight>
+
+For more information about counter and gauge aggregation functions, see the
+[hyperfunctions documentation][hyperfunctions-counter-agg].
+
+## Required arguments
+
+|Name|Type|Description
+|-|-|-|
+|`ts`|`TIMESTAMPTZ`|The time at each point|
+|`value`|`DOUBLE PRECISION`|The value at that timestamp|
+
+Only `DOUBLE PRECISION` values are accepted for the `value` parameter. For gauge
+data stored as other numeric types, cast it to `DOUBLE PRECISION` when using the
+function.
+
+<highlight type="note">
+If there are `NULL` values in your data, the aggregate ignores them. At least
+some of the values must be non-`NULL`. Otherwise, you get an error.
+</highlight>
+
+## Optional arguments
+
+|Name|Type|Description|
+|-|-|-|
+|`bounds`|`TSTZRANGE`|The largest and smallest possible times that can be input to the aggregate. Calling with `NULL`, or leaving out the argument, results in an unbounded `GaugeSummary`|
+
+<highlight type="important">
+Bounds are required for extrapolation, but not for other accessor functions.
+</highlight>
+
+## Returns
+
+|Column|Type|Description|
+|-|-|-|
+|`gauge_agg`|`GaugeSummary`|A `GaugeSummary` object that can be passed to accessor functions or other objects in the gauge aggregate API|
+
+The returned `GaugeSummary` can be used as an input to the same accessor
+functions as `CounterSummary`, except for `num_resets`.
```

Comment:
```suggestion
<highlight type="important">
The returned `GaugeSummary` can be used as an input the accessor functions
`delta`, `idelta_left`, and `idelta_right`. When this feature is mature, it will support
all the same accessor functions as `CounterSummary`, with the exception of
`num_resets`.
</highlight>
```